orca/orca-web/src/test/java/com/netflix/spinnaker/orca/FiatAuthenticationFilterFiatEnabledIT.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2025 OpsMx, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest(properties = {"spring.application.name=orcaTest", "services.fiat.enabled=true"})
+@ContextConfiguration(
+    classes = {
+      FiatAuthFilterITConfig.TestController.class,
+      FiatAuthFilterITConfig.TestSecurityConfig.class
+    })
+@AutoConfigureMockMvc
+class FiatAuthenticationFilterFiatEnabledIT {
```

Comment:
There's a lot of duplication.  Please try to combine the two test classes using e.g. @Nested.